dist/index.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
```

Comment:
should the build output be source controlled? Probably should be gitignored?

Reply:
ooh - that's weird - was ignored on my working copy...

Reply:
damn - ignore pattern was off - copied from psammead